src/app/plan/duck/selectors.ts

```diff
@@ -78,6 +71,19 @@ const getPlansWithStatus = createSelector(
         hasFailedCondition: hasMigrationError,
         latestType,
       };
+
+      //remove controller update fields
+      const { metadata } = plan.MigPlan;
+      if (metadata.annotations || metadata.generation || metadata.resourceVersion) {
```

Comment:
Just want to confirm this is basically pulling them out of the state tree in their original form, and then stripping them of these values before passing them onto the components? I don't think we want to change the k8s objects that are our state tree in any way so they purely represent the objects we got out of k8s in their original state.

Reply:
This is not mutating the original data. IF we want to be able to access these fields in the future, we can add a selector that grabs getOriginalPlans that will allow us to grab the exact object we have in the store.